src/storage/page_buffer.c

```diff
@@ -7856,28 +7859,29 @@ pgbuf_request_data_page_from_page_server (const VPID * vpid)
   /* Send a request to Page Server for the Page. */
   if (get_server_type () == SERVER_TYPE_TRANSACTION)
     {
-      constexpr size_t PAGEID_SIZE = sizeof (VPID::pageid);
-      constexpr size_t VOLID_SIZE = sizeof (VPID::volid);
-      char buffer[PAGEID_SIZE + VOLID_SIZE + sizeof (LOG_LSA)];
+      cubpacking::packer pac;
+      int size = 0;
 
-      std::memcpy (buffer, &(vpid->pageid), sizeof (vpid->pageid));
-      size_t bytes_copied = sizeof (vpid->pageid);
-
-      std::memcpy (buffer + bytes_copied, &(vpid->volid), sizeof (vpid->volid));
-      bytes_copied += sizeof (vpid->volid);
+      // *INDENT-OFF*
```

Comment:
there is an INDENT-OFF a few lines above

Reply:
did not see that, changed now, also fixed the code format in the function